qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/web/views/PortletView.groovy

```diff
@@ -54,9 +54,9 @@ class PortletView extends VerticalLayout implements AffiliationSelectionListener
         gridLayout.setSizeFull()
         //ToDo Find solution on how to best host different views in the portlet
         //gridLayout.addComponent(this.searchCustomerView)
-        gridLayout.addComponent(this.createCustomerView)
```

Comment:
where did this view go and why?

Reply:
The createCustomerView is referenced twice in the application. It gets displayed in during the create offer use case and it will be a stand alone feature. Currently, this messes up the visualisation, as the create customer view is injected as component in two different places, thus is appears twice when activated.

I am working on this currently in a separate feature branch, when these views are representing own object instances, thus they are independent.